test/boot_requirements_checker_test.rb

```diff
@@ -40,7 +40,7 @@ def find_vol(mount_point, volumes)
     let(:storage_arch) { instance_double("::Storage::Arch") }
 
     before do
-      Yast::Storage::FakeProbing.new
+      Yast::Storage::StorageManager.fake_from_yaml
```

Comment:
Hmm, calling `fake_from_yaml` without yaml is a quite hacky and unexpected usage. `fake_from_yaml` actually does two things - creating a fake instance of the StorageManager and populating `probed` with yaml.

For cases like this, in which we only want to do the first part but not the second one, we should have another method.

Like already suggested here, for example, https://github.com/yast/yast-storage/pull/193#discussion_r53643452